transform-plugins/src/main/java/io/cdap/plugin/XMLMultiParser.java

```diff
@@ -100,6 +100,12 @@ public void initialize(TransformContext context) throws Exception {
     schema = config.getSchema(collector);
     xPathExpression = config.getXPathExpression(collector);
     fieldNames = new HashSet<>();
+    builderFactory.setFeature("http://xml.org/sax/features/external-general-entities", false);
```

Comment:
should these be set based on configuration options?

Reply:
Yeah. Wanted to check if I got everything for XMLParser before modifying XMLMultiParser.